### ISSUE TITLE
team.html no longer reachable on website

### ISSUE THREAD

**Issue by Ivoz** [OpenTechSchool/www.opentechschool.org#159]
It seems when the top header was redone this page was lost?

I think it's nice to list, your thoughts @ligthyear  ?

**Comment by stefanhoth** [comment]
Team lists all organizers across the world, right? I'd like to have it back, too but there should be a text explaining what you are seeing.

**Comment by gnunicorn** [comment]
That does make sense indeed. Must have missed it when redoing the page. Also, we might want to go through the team members lists for that again....

**Comment by gsambrotta** [comment]
Hey guys we can try to bring it back, can you explain better what was this page about?
Just a list with all organizers? Or even pictures?

**Comment by xMartin** [comment]
Would that be all people from the chapter pages in one big list? Why would we do this? Mainly to make it look like a nice? Or is there another criteria who'd be on there?

**Comment by Ivoz** [comment]
Yes, just a nice page showing all the coaches and organisers that are part of opentechschool. Showing the family "together" even though we can be oceans apart.

I wouldn't disagree with adding a sentence to the page explaining it.

**Comment by xMartin** [comment]
This would need to be a client-side widget now that we're getting team data from discourse.
